src/transformers/pipelines/question_answering.py

```diff
@@ -118,7 +118,7 @@ def select_starts_ends(
         max_answer_len (`int`): Maximum size of the answer to extract from the model's output.
     """
     # Ensure padded tokens & question tokens cannot belong to the set of candidate answers.
-    undesired_tokens = np.abs(np.array(p_mask) - 1)
+    undesired_tokens = np.abs(p_mask.numpy() - 1)
```

Comment:
It will raise a value error.
`ValueError: Pipeline cannot infer suitable model classes from hf-internal-testing/tiny-random-bert`.

Reply:
Besides, temsor.numpy already used in other pipelines like [ASR](https://github.com/huggingface/transformers/blob/main/src/transformers/pipelines/automatic_speech_recognition.py#L560)

Reply:
OK, yes, looking into it we seem so assume either tf or pt everywhere in the pipeline, so even though I think this would break things for jax tensors it's not something we need to take account of at the moment. Thanks for testing!